typings/index.d.ts

```diff
@@ -0,0 +1,10 @@
+// Silence errors from libraries with no type defintions provided
+// TODO: add types to tokens and remove this line
+declare module '@thumbtack/thumbprint-tokens';
+declare module 'intersection-observer';
```

Comment:
I'm surprised the two polyfills don't have definitions on DefinitelyTyped.

Reply:
Yeah, not sure why these two don't have them